build.sbt

```diff
@@ -5,7 +5,8 @@ organization := "io.shiftleft"
 ThisBuild / scalaVersion := "2.13.5"
 // don't upgrade to 2.13.6 until https://github.com/com-lihaoyi/Ammonite/issues/1182 is resolved
 ThisBuild /Test /fork := true
-val cpgVersion = "1.3.304"
+val cpgVersion = "1.3.305"
+val cFrontendVersion = "1.3.305"
```

Comment:
make this `c2cpgVersion` instead for consistency reasons?

Reply:
That's a bit of a lie (since it applies to fuzzyc as well), but especially since we're getting rid of fuzzyc soon, sounds fine

Reply:
ahh, I see. how about two versions then?